core/backend/src/test/standalone/IModelWrite.test.ts

```diff
@@ -329,6 +331,43 @@ describe("IModelWriteTest", () => {
     rwIModel.close();
   });
 
+  it("should set a fake verifyCode for codeService that throws error for operations that affect code, if failed to open codeService ", async () => {
+    const iModelProps = {
+      iModelName: "codeServiceTest",
+      iTwinId,
+    };
+    const iModelId = await HubMock.createNewIModel(iModelProps);
+    const briefcaseProps = await BriefcaseManager.downloadBriefcase({ accessToken: "codeServiceTest", iTwinId, iModelId });
+    const originalCreateForIModel = CodeService.createForIModel;
+    // can be any errors except 'NoCodeIndex'
```

Comment:
Can you assert that we propagate the error if the error is 'NoCodeIndex' and don't set up a fake codeService?

Reply:
Added in [commit f818d0130ca3f0cbfa0125ba43c4d290563ebca9](https://github.com/iTwin/itwinjs-core/pull/6776/commits/f818d0130ca3f0cbfa0125ba43c4d290563ebca9)